### PR TITLE
Snowex 22 updates

### PR DESCRIPTION
**Pull request by jomey**
First batch of what I could find and think should move to the template.

### CODE REVIEW

File: book/tutorials/git_and_jupyterlab/jupyter.md [hunk 1]
```diff
@@ -185,7 +192,7 @@ just went with a more descriptive label instead of the default value.
 # Change to the website folder, it was created with the `clone`
 cd website
 # Add the remote
-git remote add Hackweek {{ '`{url}.git`'.format(url=website_url) }}
+git remote add Hackweek https://github.com/snowex-hackweek/website2022.git
```
Comment (scottyhq) [inline book/tutorials/git_and_jupyterlab/jupyter.md:195]: we hardcoded this due to trouble doing substitution in backticks i think, maybe we just remove the backticks?

File: book/tutorials/git_and_jupyterlab/jupyter.md [hunk 2]
```diff
@@ -198,8 +205,8 @@ list of known remotes for this repository:
 
 ```shell
  git remote -v
- Hackweek	{{ '`{url}.git`'.format(url=website_url) }} (fetch)
- Hackweek	{{ '`{url}.git`'.format(url=website_url) }} (push)
+ Hackweek	https://github.com/snowex-hackweek/website2022.git (fetch)
+ Hackweek	https://github.com/snowex-hackweek/website2022.git (push)
```
Comment (scottyhq) [inline book/tutorials/git_and_jupyterlab/jupyter.md:209]: see above, just remove backticks but keep substitution?

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 1 above)
Comment (jomey) [reply book/tutorials/git_and_jupyterlab/jupyter.md:195]: As noted in the issue, replacement is not support within code blocks in the MyST substitutions. We could just add a generic placeholder like `__GIT_REMOTE_URL__` or similar

File: book/preliminary/index.md [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 
 ## {{hackweek}} Software Carpentry Session
 
-We strongly encourage participants to review this two-day recorded fundamentals of Python and open-source workflows crash course ([Software Carpentry Schedule](swc)) in advance of the hackweek. You may choose whichever topics you'd like to brush up on or learn. Office hours in the weeks leading up the hackweek will provide an opportunity for asking questions and troubleshooting any issues you've encountered.
+We strongly encourage participants to review this two-day recorded fundamentals
+of Python and open-source workflows crash course
+([Software Carpentry Schedule](swc)) in advance of the hackweek. You may
+choose whichever topics you'd like to brush up on or learn. Office hours in the
+weeks leading up the hackweek will provide an opportunity for asking questions
+and troubleshooting any issues you've encountered.
```
Comment (JessicaS11) [inline book/preliminary/index.md:22]:
```suggestion
We strongly encourage participants to review this two-day recorded fundamentals
of Python and open-source workflows crash course
([Software Carpentry Schedule](swc)) in advance of the hackweek.
You may choose whichever topics you'd like to brush up on or learn.
Office hours in the weeks leading up the hackweek will provide an opportunity
for asking questions and troubleshooting any issues you've encountered.
```

File: book/preliminary/index.md [hunk 4]
```diff
@@ -44,16 +76,22 @@ Visit our [Introduction to Jupyter Hub](jupyterhub) page to learn more!
...
-to learn how to configure that!
+website will be done via the `git` command.
+Visit [Setting up the `git` command](git) to learn how to configure that!
 
 ### EarthData Login
 
 We'll have you download some data from NSIDC for your tutorials and projects.
-Visit our [Earthdata](earthdata) page to learn how to access and Earthdata login account if you don't already have one!
+Visit our [Earthdata](earthdata) page to learn how to access and Earthdata
+login account if you don't already have one!
```
Comment (JessicaS11) [inline book/preliminary/index.md:86]:
```suggestion
Visit our [Earthdata](earthdata) page to learn how to create an Earthdata
login account if you don't already have one!
```

File: book/preliminary/swc.md [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
+Below you will find links to our training materials from the 2021 SnowEx
+Hackweek. If you are new to Git, Linux, Python, please set aside a day or two
+to go over these materials in advance:
 
 ## Getting ready
 
-Please take a minute before the workshop to go over the [GitHub instructions](github),
-create an account.
+Please take a minute before the workshop to go over
+the [GitHub instructions](github), create an account.
```
Comment (JessicaS11) [inline book/preliminary/swc.md:19]:
```suggestion
the [GitHub instructions](github) and create an account.
```

File: book/tutorials/git_and_jupyterlab/jupyter.md [hunk 6]
```diff
@@ -38,9 +39,12 @@ do with the {{hackweek}} organization repository. A fork is also a first common
 to contribute to open-source projects and further described in the
 [Pro Git book](http://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
 
-To make a fork of the website, go to the {{ '[GitHub repository]({url})'.format(url=website_url) }} and hit the 'Fork' button in the top right corner. In the follow-up dialog, choose your user account as the destination
+To make a fork of the website, go to
+the [GitHub repository](https://github.com/snowex-hackweek/website2022) and hit
+the 'Fork' button in the top right corner. In the follow-up dialog, choose your
+user account as the destination
```
Comment (JessicaS11) [inline book/tutorials/git_and_jupyterlab/jupyter.md:46]:
```suggestion
the 'Fork' button in the top right corner.
In the follow-up dialog, choose your user account as the destination

```

File: book/tutorials/git_and_jupyterlab/jupyter.md [hunk 7]
```diff
@@ -268,18 +277,21 @@ Ping us on Slack for details on how to get your files onto there.
...
 
-![upload-file](../img/jupyterhub-upload.png)
+![upload-file](../../img/jupyterhub-upload.png)
 
 ## Summary
 
-Congratulations! That's the end of our "GitHub, Git, JupyterHub" tutorial. You now are equipped with basic `git` commands and have familiarity with the JupyterHub computing environment.
+Congratulations! That's the end of our "GitHub, Git, JupyterHub" tutorial. You
+now are equipped with basic `git` commands and have familiarity with the
+JupyterHub computing environment.
```
Comment (JessicaS11) [inline book/tutorials/git_and_jupyterlab/jupyter.md:289]:
```suggestion
Congratulations! That's the end of our "GitHub, Git, JupyterHub" tutorial.
You now are equipped with basic `git` commands and have familiarity with
the JupyterHub computing environment.
```

**Comment by jomey** [comment]
@JessicaS11 - Want to make sure I understand the intent of the suggestions. 
The added newlines will not actually show up in the rendered version as it requires two empty lines to create a new paragraph in Markdown. Are most of the suggestions then for readability?

The only rule I enforced are a maximum of 80 characters per line.

File: book/tutorials/index.md [hunk 8]
```diff
@@ -13,5 +13,4 @@ All live tutorial recordings available via UW eScience [YouTube Playlist](https:
 
 | Tutorial | Topics | Datasets |  Recording Link |
 | -  | - | - |  - |
-| [Open Science Tools](./jupyter.md) | Git, GitHub, JupyterHub | n/a | [Recording](https://youtu.be/rKhs5FLI5rc)  |
```
Comment (JessicaS11) [inline book/tutorials/index.md:16]: Sorry if I missed it - is this video now linked to elsewhere?

**Comment by JessicaS11** [comment]
> @JessicaS11 - Want to make sure I understand the intent of the suggestions.
> The added newlines will not actually show up in the rendered version as it requires two empty lines to create a new paragraph in Markdown. Are most of the suggestions then for readability?

Exactly. I was just going to leave a note to that effect. A common practice I've seen is always having a new sentence at the start of a new line, even if you also have a character limit (as you've imposed). We can take or leave that suggestion here, but I think in the long term it decreases the number of git-diff "changes" from text being updated (and limits the changes to the sentence they occur in). My OCD brain also likes having a rule that I can see (without counting characters!). ;)

**Comment by JessicaS11** [comment]
All that said, I'm excited about the updates. Especially the new figures! And I opened an issue (#114) for some duplicated content that I found (but isn't new with this PR).

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 1 above)
Comment (JessicaS11) [inline book/tutorials/git_and_jupyterlab/jupyter.md:195]:
```suggestion
git remote add Hackweek {{ '{url}.git'.format(url=website_url) }}
```

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 2 above)
Comment (JessicaS11) [inline book/tutorials/git_and_jupyterlab/jupyter.md:209]:
```suggestion
 Hackweek	{{ '{url}.git'.format(url=website_url) }} (fetch)
 Hackweek	{{ '{url}.git'.format(url=website_url) }} (push)
```

**Comment by jomey** [comment]
> Exactly. I was just going to leave a note to that effect. A common practice I've seen is always having a new sentence at the start of a new line, even if you also have a character limit (as you've imposed). We can take or leave that suggestion here, but I think in the long term it decreases the number of git-diff "changes" from text being updated (and limits the changes to the sentence they occur in). My OCD brain also likes having a rule that I can see (without counting characters!). ;)

Sorry for jumping the gun here. Feel free to just commit them and merge as needed (including the PR). I probably won't get to this until end of next week.

Regards character limits - I am a big fan of editors that support the `.editorconfig`. Probably a good enforcement rule to add (and I have it set locally for all my docs). :smile:

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 1 above)
Comment (jomey) [reply book/tutorials/git_and_jupyterlab/jupyter.md:195]: See the above discussion:
https://github.com/uwhackweek/jupyterbook-template/pull/113#discussion_r965595340

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 2 above)
Comment (jomey) [reply book/tutorials/git_and_jupyterlab/jupyter.md:209]: Same here https://github.com/uwhackweek/jupyterbook-template/pull/113#discussion_r965595340

**Comment by scottyhq** [comment]
> A common practice I've seen is always having a new sentence at the start of a new line, even if you also have a character limit 

I actually prefer seeing paragraphs in the text editor! Can't please everyone with the formatting rules and .editorconfig catches a lot of git diff excess, but doesn't have a way of enforcing line length limits. To truly enforce character limits I think we'd have to use precommit and an autoformatter, but i'm not really in favor of adding more things to keep track of

**Review by scottyhq — APPROVED** [review]

**Comment by scottyhq** [comment]
thanks for doing this @jomey if you want to add any of the suggested commits go for it, overall it's great to get these changes in and we can followup on the backticks and other existing linked issues

File: book/tutorials/git_and_jupyterlab/jupyter.md (hunk 2 above)
Comment (jomey) [reply book/tutorials/git_and_jupyterlab/jupyter.md]: Resolved through a suggestion found by @JessicaS11 below.

**Pull request merged**